### ISSUE TITLE
(Vote) Hardcore mode?

### ISSUE THREAD

**Issue by Lemmmy** [Lemmmy/CCJam-2016#51]
Ability for users to make their account hardcore - lose everything on death (an account option)

Vote with :+1: or :-1: as a reaction

**Comment by ry00000** [comment]
I want Hardcore mode. All my props belong to this.

**Comment by Lemmmy** [comment]
do you know how to vote on a github issue

**Comment by Lemmmy** [comment]
![](http://actually.a.shitty.download/companion_guy_ambien.png)

these things
its a neat github feature
you should give it
a
try

**Comment by ry00000** [comment]
Yes I know how to use them I just wanted to share my 2 cents on this.

**Comment by CrazedProgrammer** [comment]
yes
